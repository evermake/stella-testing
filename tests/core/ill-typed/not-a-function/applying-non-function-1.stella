language core;

fn main(f : Nat) -> Nat {
  return f(f);
}