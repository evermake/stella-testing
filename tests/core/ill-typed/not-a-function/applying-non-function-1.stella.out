An error occurred during typechecking!
Type Error Tag: [ERROR_NOT_A_FUNCTION]
expected a function type but got
  Nat
for the expression
  f
in the function call
  f (f)